doc/api/process.md

```diff
@@ -793,15 +816,16 @@ if (process.getuid) {
 }
 ```
 
-## process.hrtime()
+## process.hrtime([previousValue])
 
 Returns the current high-resolution real time in a `[seconds, nanoseconds]`
 tuple Array. It is relative to an arbitrary time in the past. It is not
 related to the time of day and therefore not subject to clock drift. The
 primary use is for measuring performance between intervals.
 
-You may pass in the result of a previous call to `process.hrtime()` to get
-a diff reading, useful for benchmarks and measuring intervals:
+The result of a previous call to `process.hrtime()` can be passed as the
+argument to the function, to get a diff reading, which can be useful for
+benchmarks and measuring intervals:
```

Comment:
This doc change should be in a different commit. It's unrelated to the `cpuUsage()` change. If the later needs to be reverted don't want to loose the former.